larsim/MCSTReco/MCTrackRecoAlg.cxx

```diff
@@ -184,11 +186,9 @@ namespace sim {
           // Add in a voxel before and after to account for MCSteps
           // the line distance allows for 1mm GEANT multiple columb scattering correction,
           // small compared to average MCStep-to-MCStep distance
-          if ((a * edep.pos._x + b * edep.pos._y + c * edep.pos._z + d) /
-                  sqrt(pow(a, 2) + pow(b, 2) + pow(c, 2)) <=
+          if ((a * edep.pos.X() + b * edep.pos.Y() + c * edep.pos.Z() + d) / std::hypot(a, b, c) <=
                 dist + 0.03 &&
```

Comment:
See above re literals.

Reply:
Agree, but won't fix.

Reply:
See above re suggested TODO.

Reply:
I have added a `@todo` in the class Doxygen documentation, since this is an issue across its code.